obs-mcp/evals/system.yaml

```diff
@@ -0,0 +1,137 @@
+
+    "ragas:faithfulness":
+      threshold: 0.8
+      description: "How faithful the response is to the provided context"
+      default: false # By default the value is false
+
+    # Ragas Context/Retrieval Evaluation metrics
+    "ragas:context_recall":
+      threshold: 0.8
+      description: "Did we fetch every fact the answer needs?"
+
+    "ragas:context_precision_with_reference":
+      threshold: 0.7
+      description: "How precise the retrieved context is (with reference)"
+
+    "ragas:context_precision_without_reference":
+      threshold: 0.7
+      description: "How precise the retrieved context is (without reference)"
+
+    "ragas:context_relevance":
+      threshold: 0.7
+      description: "Is what we retrieved actually relevant to user query?"
+
+    # Custom metrics
+    "custom:answer_correctness":
+      threshold: 0.75
+      description: "Correctness vs expected answer using custom LLM evaluation"
+
+    "custom:tool_eval":
+      description: "Tool call evaluation comparing expected vs actual tool calls"
+
+  # Conversation-level metrics metadata
+  conversation_level:
+    # DeepEval metrics
+    "deepeval:conversation_completeness":
+      threshold: 0.8
+      description: "How completely the conversation addresses user intentions"
+      default: false
+
+    "deepeval:conversation_relevancy":
+      threshold: 0.7
+      description: "How relevant the conversation is to the topic/context"
+
+    "deepeval:knowledge_retention":
+      threshold: 0.7
+      description: "How well the model retains information from previous turns"
+
+# Output Configuration
+output:
+  output_dir: "./eval_output"
+  base_filename: "evaluation"
+  enabled_outputs: # Enable specific output types
+    - csv # Detailed results CSV
+    - json # Summary JSON with statistics
+    - txt # Human-readable summary
+
+  # CSV columns to include
+  csv_columns:
+    - "conversation_group_id"
+    - "turn_id"
+    - "metric_identifier"
+    - "score"
+    - "threshold"
+    - "result"
+    - "reason"
+    - "query"
+    - "response"
+    - "execution_time"
+
+# Visualization settings
+visualization:
+  figsize: [12, 8] # Graph size (width, height)
+  dpi: 300 # Image resolution
+
+  # Graph types to generate
+  enabled_graphs:
+    - "pass_rates" # Pass rate bar chart
+    - "score_distribution" # Score distribution box plot
+    - "conversation_heatmap" # Heatmap of conversation performance
+    - "status_breakdown" # Pie chart for pass/fail/error breakdown
+
+# Environment Variables - Automatically get set before any imports
+environment:
+  DEEPEVAL_TELEMETRY_OPT_OUT: "YES" # Disable DeepEval telemetry
+  DEEPEVAL_DISABLE_PROGRESS_BAR: "YES" # Disable DeepEval progress bars
+
+  LITELLM_LOG: ERROR # Suppress LiteLLM verbose logging
+
+# Logging Configuration
+logging:
+  # Source code logging level
+  source_level: INFO # DEBUG, INFO, WARNING, ERROR, CRITICAL
```

Comment:
when it fails there is not much useful info in INFO, maybe default to DEBUG? update: nah even debug does not log tool call traces so probably need to take it to lightspeed-evaluation